infra/gravity/cluster_status.go

```diff
@@ -71,50 +71,41 @@ func (c *TestContext) CollectLogs(prefix string, nodes []Gravity) error {
 	ctx, cancel := context.WithTimeout(c.ctx, c.timeouts.CollectLogs)
 	defer cancel()
 
-	nodes, err := c.reorderNodesForCollection(ctx, nodes)
-	if err != nil {
-		c.Logger().WithError(err).Warn("Failed to reorder nodes for collection.")
-		// nodes is still valid
-	}
-
 	c.Logger().WithField("nodes", nodes).Debug("Collecting logs from nodes.")
-	firstNodeArgs := []string{"--filter=system", "--filter=kubernetes"}
-	nodeArgs := []string{"--filter=system"}
-	err = c.collectLogsFromNodes(ctx, nodes, prefix, firstNodeArgs, nodeArgs)
-	return trace.Wrap(err)
-}
 
-func (c *TestContext) reorderNodesForCollection(ctx context.Context, nodes []Gravity) ([]Gravity, error) {
 	api, other, err := apiserverNode(ctx, nodes)
-	if err != nil {
-		// Return nodes unaltered in case of error
-		return nodes, trace.Wrap(err)
+	if err != nil || api == nil {
+		c.Logger().WithError(err).Warn("Unable to determine api-server")
```

Comment:
nit: We tend to stick to convention of ending the log messages with a dot.

Reply:
Thanks.  Fixed in d55f1c9.